domain/policy.go

```diff
@@ -182,15 +182,30 @@ type Requirement struct {
 
 // Policy is the approval policy configuration
 type Policy struct {
-	ID           string            `json:"id" yaml:"id" validate:"required"`
-	Version      uint              `json:"version" yaml:"version" validate:"required"`
-	Description  string            `json:"description" yaml:"description"`
-	Steps        []*Step           `json:"steps" yaml:"steps" validate:"required,min=1,dive"`
-	Requirements []*Requirement    `json:"requirements,omitempty" yaml:"requirements,omitempty" validate:"omitempty,min=1,dive"`
-	Labels       map[string]string `json:"labels,omitempty" yaml:"labels,omitempty"`
-	IAM          *IAMConfig        `json:"iam,omitempty" yaml:"iam,omitempty" validate:"omitempty,dive"`
-	CreatedAt    time.Time         `json:"created_at,omitempty" yaml:"created_at,omitempty"`
-	UpdatedAt    time.Time         `json:"updated_at,omitempty" yaml:"updated_at,omitempty"`
+	ID           string             `json:"id" yaml:"id" validate:"required"`
+	Version      uint               `json:"version" yaml:"version" validate:"required"`
+	Description  string             `json:"description" yaml:"description"`
+	Steps        []*Step            `json:"steps" yaml:"steps" validate:"required,min=1,dive"`
+	Appeal       PolicyAppealConfig `json:"appeal" yaml:"appeal" validate:"omitempty"`
```

Comment:
should this be `*PolicyAppealConfig`? because the existing policies are not having this field

Reply:
The `gorm` migration will autofill the table with a default value for this column when the column gets added.
But given that migration will be done by `go-migrate` in the latest master. Will check this out

Reply:
Done. made the suggested change.